test/e2e/framework/framework.go

```diff
@@ -143,6 +144,10 @@ func (f *Framework) CommonLabels() map[string]string {
 	}
 }
 
+func (f *Framework) GetDefaultScyllaCluster() *scyllav1.ScyllaCluster {
+	return TestContext.DefaultScyllaCluster.DeepCopy()
```

Comment:
Why would you do it this way? I wanted to avoid rendering it more than once for the entire suite (https://github.com/scylladb/scylla-operator/pull/1508/files/8031e49a51c0e3ef31c83e88fa38730d9e2ffb5f#r1372055888).

Reply:
copy of https://github.com/scylladb/scylla-operator/pull/1508#discussion_r1371907162, let's resolve there